apps/core-frontend/src/ui/fixedrates/FixedRatesListView.tsx

```diff
@@ -25,7 +25,11 @@ export function FixedRatesListView(
   // TODO: Implement custom sorting UI?
   const sortedPrincipalTokenInfos = useMemo(() => {
     return [...openPrincipalTokenInfos]
-      .sort((info) => info.extensions.createdAtTimestamp)
+      .sort(
+        (info1, info2) =>
+          info1.extensions.createdAtTimestamp -
+          info2.extensions.createdAtTimestamp,
+      )
       .reverse();
```

Comment:
I just realized we're sorting and reversing which doesn't make sense. We should remove the `reverse()` and just return `info2... - info1`.

Reply:
good catch